handler/implementations/kernelLastCap.go

```diff
@@ -69,23 +73,13 @@ func (h *KernelLastCapHandler) Open(
 		return fuse.IOerror{Code: syscall.EACCES}
 	}
 
```

Comment:
As this deviates from our usual approach of opening/closing the fds across all our handlers, please add a note here to explain why we are allowing it in this particular case. Something like ...

"In this particular handler we are not opening the FS node as there is no need to validate the process credentials to access this resource (i.e. /proc/sys/kernel/cap_last_cap is read-only for all processes in the system). This is done for optimization purposes to save cycles ... blah blah ..."

Reply:
Talked with Rodny and we agreed that what the cap_last_cap handler is doing should be the default action for most handlers, so I'll skip the comment.